content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -72,19 +72,23 @@ export class ContextMenuItem extends React.PureComponent {
     super(props);
     this.onClick = this.onClick.bind(this);
     this.onKeyDown = this.onKeyDown.bind(this);
-    this.focusFirst = this.focusFirst.bind(this);
   }
 
   onClick() {
     this.props.hideContext();
     this.props.option.onClick();
   }
 
-  focusFirst(button) {
-    if (button) {
-      button.focus();
-    }
-  }
+  // This function focuses the first option when the menu is rendered.
```

Comment:
Can you remove this instead of commenting it out? You can reference the code in the bug to restore/fix the isssue if you want to save it

Reply:
okay, no problem!